src/main/java/ch/njol/skript/expressions/ExprChatRecipients.java

```diff
@@ -40,71 +40,70 @@
 import ch.njol.util.coll.CollectionUtils;
 
 @Name("Chat Recipients")
-@Description("Recipients of chat events where this is called.")
+@Description("Recipients of chat/broadcast events where this is called.")
 @Examples("chat recipients")
-@Since("2.2-Fixes-v7, 2.2-dev35 (clearing recipients)")
-public class ExprChatRecipients extends SimpleExpression<Player> {
+@Since("2.2-Fixes-v7, 2.2-dev35 (clearing recipients), INSERT VERSION (broadcast event)")
+public class ExprChatRecipients extends SimpleExpression<CommandSender> {
 
 	static {
-		Skript.registerExpression(ExprChatRecipients.class, Player.class, ExpressionType.SIMPLE, "[chat][( |-)]recipients");
-	}
-
-	@Override
-	public boolean isSingle() {
-		return false;
-	}
-
-	@Override
-	public Class<Player> getReturnType() {
-		return Player.class;
-	}
-
-	@Override
-	public Class<?>[] acceptChange(final ChangeMode mode) {
-		return CollectionUtils.array(Player[].class);
+		Skript.registerExpression(ExprChatRecipients.class, CommandSender.class, ExpressionType.SIMPLE, "[chat][( |-)]recipients");
 	}
 
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
-		if (!(getParser().isCurrentEvent(AsyncPlayerChatEvent.class))) {
-			Skript.error("Cannot use chat recipients expression outside of a chat event", ErrorQuality.SEMANTIC_ERROR);
+		if (!(getParser().isCurrentEvent(AsyncPlayerChatEvent.class, BroadcastMessageEvent.class))) {
+			Skript.error("Cannot use chat recipients expression outside of a chat or server broadcast event", ErrorQuality.SEMANTIC_ERROR);
```

Comment:
```suggestion
			Skript.error("Cannot use chat recipients expression outside of a chat or server broadcast event");
```

Reply:
Existing implementation

Reply:
I think it's okay to edit it while you're at it, it's not much of work.